src/BasketBundle/Form/PaymentType.php

```diff
@@ -99,7 +100,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $sub = $builder->create('paymentMethod', $choiceType, array(
             'expanded' => true,
-            'choice_list' => new SimpleChoiceList($choices),
+            'choice_list' => function (Options $opts, $previousValue) use ($choices) {
```

Comment:
You are right... is it ok if I do something like:        
 if (interface_exists('Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface')) { 
//choice_loader
}else{
//choice_list
}